### ISSUE TITLE
Import module from different working directories

### ISSUE THREAD

**Issue by paulina-t** [jerabaul29/example_python_package#1]
How can I import example_package from another directory, for instance, some_sub_module? @jerabaul29

**Comment by jerabaul29** [comment]
I am not sure of what you mean. You can import the module from any directory, as it is globally available on the machine once the PYTHONPATH is set. Do you have an example of what you try, and how it fails?

**Comment by jerabaul29** [comment]
For example, this works fine if your paths are set correctly:

```
~$ z examp
~/Desktop/Git/example_python_module [main|✔]> cd example_package/some_sub_module/
~/Desktop/Git/example_python_module/example_package/some_sub_module [main|✔]> python3
Python 3.8.5 (default, Jul 28 2020, 12:59:40) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import example_package
>>> from example_package.some_other_sub_module import some_other_sub_module_1
>>> some_other_sub_module_1.some_other_sub_module_2_hello()
hello from some_other_sub_module_2
```

**Comment by paulina-t** [comment]
Thansk! That works, actually! 

However, I was trying to run it in Spyder, and got this error:

```
Python 3.8.5 (default, Sep  4 2020, 07:30:14) 
Type "copyright", "credits" or "license" for more information.

IPython 7.19.0 -- An enhanced Interactive Python.

from os import getcwd

getcwd()
Out[2]: '/home/paulinast/Desktop/repos/example_python_module/example_package/some_sub_module'

import example_package
Traceback (most recent call last):

  File "<ipython-input-3-c0c496339dc8>", line 1, in <module>
    import example_package

ModuleNotFoundError: No module named 'example_package'
``` 
Do you know why?

**Comment by jerabaul29** [comment]
Do you use a "special" (i)python kernel, which may be different from the system wide install? A bit like:

https://stackoverflow.com/questions/12204981/ipython-has-different-sys-path-than-python-and-does-not-use-pythonpath

See also discussions there:

https://stackoverflow.com/questions/43985787/how-to-make-jupyter-notebook-use-pythonpath-in-system-variables-without-hacking

Another thing is that you may need to add some extra ```__main__.py``` in all sub dirs depending on what kind of python kernel you use (I think).

**Comment by paulina-t** [comment]
Great suggestion! I found out that I need to add the path using the PYTHONPATH manager in Spyder. See:

http://physicalmodelingwithpython.blogspot.com/2015/10/paths-in-python.html

I tried another workaround as mentioned here:

http://physicalmodelingwithpython.blogspot.com/2015/10/paths-in-python.html

but it didn't work.

Good to know that the problem wasn't related to the `__init__.py` files, though!
